packages/toolpad-app/src/components/EditableText.tsx

```diff
@@ -120,7 +128,7 @@ const EditableText = React.forwardRef<HTMLInputElement, EditableTextProps>(
             overflow: 'hidden',
             whiteSpace: 'nowrap',
             fontSize: theme.typography[typographyVariant].fontSize,
-            height: theme.typography[typographyVariant].fontSize,
+            height: `calc(${theme.typography[typographyVariant].fontSize} + 2px)`,
```

Comment:
Isn't this what `em` units are for?

Reply:
`em` is based on the font size of the parent i think? so might work here but not sure.
also this could probably be written as `${theme.typography[typographyVariant].fontSize + 2 }px` if it sounds preferrable not to use the `calc()`

Reply:
> `em` is based on the font size of the parent i think?

Yes for properties like `font-size`. For other properties like `height` it's based on the fontsize of the element itself